packages/ui-components/src/components/tag-status/tag-status.scss

```diff
@@ -0,0 +1,43 @@
+@import '../../assets/styles/globals';
+
+.status {
+	&-container {
+		display: flex;
+		align-items: center;
+
+		kv-icon {
+			--icon-width: 16px;
+			--icon-height: 16px;
+		}
+	}
+
+	&-success {
+		kv-icon {
+			--icon-color: #{kv-color('success')};
+		}
+	}
+
+	&-error {
+		kv-icon {
+			--icon-color: #{kv-color('error')};
+		}
+	}
+
+	&-unknown {
```

Comment:
I think here should be `-none` but please confirm

Reply:
Yeap :heavy_check_mark: